katdal/ms_extra.py

```diff
@@ -44,6 +44,18 @@
         casacore_binding = 'pyrap'
     except ImportError:
         casacore_binding = ''
+    else:
+        # Perform python-casacore version checks
```

Comment:
Wouldn't it be easier to check `casacore.__version__`?

Reply:
I poked around for it, but it doesn't exist.

Reply:
```python
>>> import casacore
>>> casacore.__version__
'2.2.1'
```
Works for me. Definitely also exists in 2.1.2, haven't checked older version. I'm always a bit distrustful of `get_distribution` because it reports what pip remembers installing, which isn't always the same as what's being picked up from the Python path.

Reply:
Apologies, I should have read more carefully. I've updated the PR to use `casacore.__version__`.

Reply:
Exactly as Bruce said. That's why we have a little package called `katversion` :-)